install/spoolman-install.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/Donkie/Spoolman
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
```

Comment:
Don't install unnecessary dependencies. Removed `--no-install-recommends`, `unzip`, `jq` and `git`
```
$STD apt-get install -y \
  build-essential \
  curl \
  sudo \
  make \
  libpq-dev \
  gpg \
  ca-certificates \
  mc
```

Reply:
unzip is needed, mc dont have unzip, the others have been removed

"mc (Midnight Commander) doesn't have a direct dependency on unzip in Debian.
mc has built-in functionality to unpack and navigate ZIP archives.
unzip is a separate tool for extracting ZIP archives with more features."

Reply:
It does when `--no-install-recommends` is removed